src/resources/cli/defs.bzl.override

```diff
@@ -51,7 +78,7 @@ CLASS_JAR=$2
 OUTPUT_FILE=$3
 
 echo '[' > $OUTPUT_FILE
-json_files=$($ZIPPER v $CLASS_JAR | grep -oE '[^ ]+-symbols.json$')
+json_files=$($ZIPPER v $CLASS_JAR | grep -oE '[^ ]+-symbols.json$' || echo '')
```

Comment:
I'm not entirely sure this is valid yet, but I ran into some conditions in which there was no symbol file and this change causes us to "fail gracefully". Some investigation may be warranted on the scenario that leads to this.